staging/src/k8s.io/kube-aggregator/pkg/apiserver/handler_discovery_test.go

```diff
@@ -37,9 +37,21 @@ import (
 	scheme "k8s.io/client-go/kubernetes/scheme"
 	"k8s.io/client-go/tools/cache"
 	apiregistrationv1 "k8s.io/kube-aggregator/pkg/apis/apiregistration/v1"
-	"k8s.io/kube-aggregator/pkg/apiserver"
 )
 
+func newDiscoveryManager(rm discoveryendpoint.ResourceManager) *discoveryManager {
+	return NewDiscoveryManager(rm).(*discoveryManager)
```

Comment:
Something to think about for a followup: why does this package need both an interface and an implementation struct? Why not just an exported implementation struct?